apps/hobbies-helsinki/README.md

```diff
@@ -8,16 +8,68 @@ Production server: https://harrastukset.hel.fi
+└─────────────┘               └───────────────┘
+```
+
+### Headless CMS datasources
+
+- Test / Staging: https://harrastus.hkih.stage.geniem.io/graphql ([Wordpress-admin](https://harrastus.hkih.stage.geniem.io/wp-admin/))
+- Production: https://harrastus.content.api.hel.fi/wp-admin/graphql ([Wordpress-admin](https://harrastus.content.api.hel.fi/wp-admin/wp-admin/))
+
+### Route (pages)
+
+The pages are served with some server side rendering (SSR) mechanism to offer better search engine optimization (SEO) and fast user friendly UI.
+The pre-rendering process that we mostly use here is SSG - "automatically generated as static HTML + JSON (uses getStaticProps)".
+More about NextJS's data fetching in https://nextjs.org/docs/basic-features/data-fetching/overview.
+
+There are some rules that a content manager must follow while maintaining the CMS:
+
+1. **/ (The root path / the front page)**: The root path is set as a landing page and it acts as a home page or the front page. The slug for this page must be an empty string, so the uri is `/` in every language.
+2. **/search (The search page)**: The search page gets no content from the CMS, except the SEO information. The uri for this page must be `/search` in every language. It is translated to different languages with the `i18nRoutes.config.json` configuration file.
+3. **/article-archive (The article archive page)**: The article archive page gets the SEO content from the CMS and also makes some hardcoded article search queries there. The page is mostly rendered with the components that the HCRC-lib offers. The uri must be `/article-archive` in every language. It is translated to different languages with the `i18nRoutes.config.json` configuration file.
+4. **/pages (The root for the dynamic pages)**: All the dynamic pages must be children of the `/pages` -root-page. This strategy must be followed so that the application can internally handle the dynamic CMS pages. The path is translated to different languages with the `i18nRoutes.config.json` configuration file.
+
 ## Developing locally
 
 Run the development server:
 
 ```
 yarn dev
+# or
+docker-compose up
 ```
 
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
 
+## Setting up Husky pre-commit hooks:
+
+1. Run `yarn install` and `yarn postinstall` on project root.
+2. Try `git commit -m foo`. It does not commit anything for real but pre-commit hook should be triggered.
+
+## Available scripts
+
+###`yarn dev`
+
+Runs the application in development mode.
+Open [http://localhost:3000](http://localhost:3000) to view in the browser.
+
+The page will reload if you make changes.
+
+###`yarn build`
+
+Builds the production application in the `.next` folder.
+Production build can be run locally with `yarn start`.
+
+###`yarn test`
+Launches the test runner in the interactive watch mode.
+
```

Comment:
There is lot of other scripts that could be also listed like lint, typecheck, etc.

Reply:
added some